src/components/context-menu-component.ts

```diff
@@ -0,0 +1,39 @@
+import * as html from '@hyperapp/html'
+import {getLogger} from './../logger'
+
+const logger = getLogger('main')
+
+export const state = {
+  menuOpen: false,
+}
+
+export const actions = {
+  toggleMenu: ({menuOpen: !state.menuOpen}),
+}
+
+export function view(
+  _state: any,
+  _actions: any,
+  items: Map<string, () => void> = new Map([
+    ['New text node', function onItem1() {logger.log('item 1 clicked')}],
+    ['New handwritten node', function onItem2() {logger.log('item 2 clicked')}],
+    ['Delete node', function onItem3() {logger.log('item 3 clicked')}],
+  ]),
+) {
+  return html.div({

```

Comment:
please indent/format as the other files have done for the view function html elements